ci/update-deps.js

```diff
@@ -63,7 +63,7 @@ async function checkVersionExists(plugin, version) {
 
 async function findPatch(plugin, minor) {
     // TODO: this is dumb, and will likely need to be changed when we add next dependency that doesn't follow the semver pattern
-    let currentPatch = plugin === 'parsely' ? '0' : 'beta';
+    let currentPatch = plugin === 'jetpack' ? 'beta' : '0';
```

Comment:
@rinatkhaziev Added a small change here so that semantic versioning for [vip-block-data-api's releases](https://github.com/Automattic/vip-block-data-api/releases/) will also work. Without this, the code was only checking for `1.0` and `1.0-beta`, skipping the available `1.0.1`. Jetpack seems to be the only one using the `-betaX` format so I changed it so that semver is default.